test/sync/test_worker.js

```diff
@@ -34,9 +34,10 @@ module.exports = {
     q.get.onThirdCall().yields(null, null);
     q.get.onCall(3).yields(null, task2);
     q.ack.yields(new Error('test ack failed'));
-    var worker = new Worker(q, processor, metrics, {interval: 10});
+    var worker = new Worker(q, processor, metrics, {interval: 10, backoff: {strategy: 'none'}});
```

Comment:
Is 'none' a special value, or just depending on it not matching any known strategy?